### ISSUE TITLE
New data set?

### ISSUE THREAD

**Issue by npaterno** [hadley/babynames#36]
Hi Hadley, 

I work for OpenIntro managing a small team of volunteers who build datasets to be included in packages associated with those textbooks. One of my volunteers built a dataset using the same source as this package. It includes the top 20 names for all years up to 2020. 

I wanted to raise an issue to see if you'd be interested in adding it to this package before I opened a PR. You can view the dataset [here](https://github.com/npaterno/data_hunters/blob/master/R/data-baby_names.R). I can change the name of the dataset to better reflect the content if you decide its worth adding to babynames; it was named by the volunteer with the intent of being in openintro but seems more relevant here.

Best, 
~Nick

**Comment by arvi1000** [comment]
I agree it would be useful to have the latest data (2024 is now available)
